prime-router/settings/readme.md

```diff
@@ -0,0 +1,5 @@
+Organization files have settings for organizations, senders and receivers. There are two types of organization files:
```

Comment:
I'd be interested in @mauricereeves thoughts on this as well.   Excepting emergencies, I think its wise to have two reviewers of all Prod settings changes, since they are untestable.  The PR process seems well suited for that.   OTOH the whole point of the settings API was to be able to do settings changes without having to wait for deployments.

I think this meets both requirements:   Lets say you are tweaking VT-DOH.   You pull the existing setting from the Prod API, you tweak it, you put your change here, pop out a quick PR, it gets reviewed, then you manually load that setting yml into Prod.   So the .yml files here merely act as documentation - the file is not "used" at all during any deployment.

Some tweaks in the process might be needed
- to make it easier to compare the new to the existing.
- I predict we'll have orgs that don't want their info so public; we'll need a separate private repo for the "Jurisdiction Operations and Onboarding Subteam" (The JOOS, and yes I just made this up)